CODEOWNERS

```diff
@@ -1,14 +1 @@
-*   @kevinbarabash @jeresig @somewhatabstract @jandrade @github-actions[bot]
-packages/wonder-blocks-breadcrumbs/*   @jandrade @kevinbarabash @jeresig
-packages/wonder-blocks-button/*   @kevinbarabash @jeresig
-packages/wonder-blocks-color/*   @kevinbarabash @jeresig
-packages/wonder-blocks-core/*   @somewhatabstract @kevinbarabash @jeresig
-packages/wonder-blocks-data/*   @somewhatabstract @jeresig
-packages/wonder-blocks-icon-button/*   @kevinbarabash @jeresig
-packages/wonder-blocks-link/*   @kevinbarabash @jeresig
-packages/wonder-blocks-modal/*   @jandrade @kevinbarabash @jeresig
-packages/wonder-blocks-popover/*   @jandrade @jeresig
-packages/wonder-blocks-testing/*   @somewhatabstract @jandrade @jeresig
-packages/wonder-blocks-timing/*   @somewhatabstract @jeresig
-packages/wonder-blocks-tooltip/*   @somewhatabstract @kevinbarabash @jeresig
-packages/wonder-blocks-typography/*   @briangenisio @kevinbarabash @jeresig
+*   @Khan/wonder-blocks
```

Comment:
Out of curiosity... do you think we still need this file now that we use Gerald to notify the team?

Reply:
I don't know - I guess we may not need it! We could just rely on Gerald... maybe I'll just remove this file instead!